### DISCUSSION TITLE
Use Docusaurus CSS in VSCode markdown preview, is it possible?

### DISCUSSION THREAD

**Discussion by softshape** [facebook/docusaurus#5673] [Q&A] [answered]
Hi all,

Visual Studio Code already has a markdown live preview feature, but with its own style. VSCode can use an external CSS for that, but does Docusaurus provide a single CSS file with can be used for markown preview in standard style?

**Comment by slorber** [comment, marked as answer]
Docusaurus theme classic uses Infima.dev as CSS framework 

I don't think we want to invest much time creating a pixel-perfect MD IDE preview, particularly when it would not even work for MDX / React components. You would have to work on the integration by yourself or demonstrate strong interest from the community.
